src/NodeVisitor/FullyQualifiedNamespaceUseScoperNodeVisitor.php

```diff
@@ -40,7 +40,7 @@ public function enterNode(Node $node)
             && false === ($node->hasAttribute('phpscoper_ignore')
             && true === $node->getAttribute('phpscoper_ignore'))
         ) {
-            return new Name(Name::concat($this->prefix, (string) $node));
+            return new Name('\\'.Name::concat($this->prefix, (string) $node));
```

Comment:
Casting Node to string omits `\` at start so the new Node isn't registered as fully qualified.